GitHub-Test-DHprojects/GitHub-Test-4-PyNLPDiscuss/lyndonDiscussion.md

```diff
@@ -0,0 +1,23 @@
+
+# Lyndon's Discussion Post
+
+
+**A particular method of NLP that I found interesting was the idea of topic modeling. The idea of locating "hidden themes" with text documentaries in order to label them automatically.**
+
+As I have previous mentioned, topic modeling for documents sounds like an interesting method to try out during a class assignment or project.
+
+[Bee Movie Script](http://www.script-o-rama.com/movie_scripts/a1/bee-movie-script-transcript-seinfeld.html Links to an external site.I feel that using this script would be an interesting test to see what category this would be listed under.)
+
+```
+{
+# Lyndon's Discussion Post
+
+
+**A particular method of NLP that I found interesting was the idea of topic modeling. The idea of locating "hidden themes" with text documentaries in order to label them automatically.**
+
+As I have previous mentioned, topic modeling for documents sounds like an interesting method to try out during a class assignment or project.
+
+[Bee Movie Script](http://www.script-o-rama.com/movie_scripts/a1/bee-movie-script-transcript-seinfeld.html Links to an external site.I feel that using this script would be an interesting test to see what category this would be listed under.)
```

Comment:
Hey I kinda wanna do the same thing with 80s scripts. Could be a cool combo of movie scripts. Maybe all Pixar movies?